Logging/Log.cs

```diff
@@ -140,6 +140,25 @@ public static void Error(string format, params object[] args)
         }
 
         /// <summary>
+        /// Log trace with format string
+        /// </summary>
+        public static void TraceFormat(string traceFormat, params object[] args)
```

Comment:
In addition, I would like to see us not have to prepend ClassName.MethodName(): everywhere we do logging, the logging infrastructure could easily resolve this, see `Log.Error` -- these changes are certainly not required as part of this PR, just mentioning for posterity.